package.json

```
@@ -55,8 +55,8 @@
         "@mui/material": "^5.4.1",
         "@mui/system": "^5.4.1",
         "framer-motion": "^4.1.17",
-        "react": "^17.0.2",
-        "react-dom": "^17.0.2"
+        "react": "^18.0.0",
```

Retain React 17 compatibility.
```suggestion
        "react": "^17.0.2 || ^18.0.0",
```